_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,405 @@
+
+```html
+<p aria-hidden="true">
+	This circle is nice.
+	<span style="height: 25px;
+		     width: 25px;
+		     background-color: #bbb;
+		     border-radius: 50%;
+		     display: inline-block;">
+	</span>
+</p>
+```
+
+#### Passed Example 13
+
+This parapgraph holds the visual reference word "right". The user is told to find the navigation on the right and the navigation is also correctly identified by a `nav` element.
+
+```html
+<head>
+	<title>Passed example 13 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This parapgraph holds the visual reference word "right". The user is told to find the menu on the right but the menu is not identified in any other way.
+
+```html
+<head>
+	<title>Failed example 1 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+			</ul>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 2
+
+This parapgraph holds the visual reference word "right". The user is told to find the navigation on the right and the navigation is correctly identified by a `nav` element, but there are 2 `nav` elements on the page so the user doesn't know which one to use.
+
+```html
+<head>
+	<title>Failed example 2 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<nav>
+		<ul>
+			<li>
+				<a href="https://www.w3.org/">W3C homepage</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/standards/">Standards</a>
+			</li>
+		</ul>
+	</nav>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Failed Example 3
+
+This parapgraph holds the visual reference word "triangle" and no other indication is present so the rule fails. The fact that the triangle menu is on a different page of the same website does not restrict the rule. 
```

Comment:
Maybe add that the word "triangle" is **not** present on the destination page (to emphasise difference with Passed Example 8, given that they only differ in the destination of the link it might be a bit hard to see it at first).